node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -58,39 +58,50 @@
     }
 }
 
-/* MediaCore preview */
-
-#filepreview-container #filepreview-mediacore,
+#filepreview-container #filepreview-video-container,
 #filepreview-container #filepreview-markdown-container {
     padding: 20px 0;
 }
 
-#filepreview-container #filepreview-mediacore iframe {
-    width: 100%;
-}
-
-.mobile #filepreview-container #filepreview-mediacore iframe {
-    width: 340px; /* iOS will need an explicit width for the iframe */
-}
-
-#filepreview-container #filepreview-mediacore.filepreview-mediacore-audio iframe {
-    height: 24px;
-}
+/* Video preview */
 
 @media (min-width: 780px) {
-    #filepreview-container #filepreview-mediacore iframe {
+    #filepreview-container #filepreview-video-container {
         height: 600px;
     }
 }
 
 @media (min-width: 400px) and (max-width: 780px) {
-    #filepreview-container #filepreview-mediacore iframe {
+    #filepreview-container #filepreview-video-container {
         height: 400px;
     }
 }
 
 @media (max-width: 400px) {
-    #filepreview-container #filepreview-mediacore iframe {
+    #filepreview-container #filepreview-video-container {
         height: 200px;
     }
 }
+
+/* Video.js overrides. These are required to align the video
+ element with the comment widget and to get it to resize correctly */
+
+.video-js {
+    background-color: #FFF;
+    position: unset;
+}
+
+#filepreview-video-container .video-js .vjs-control-bar {
+    margin: 0 20px;
```

Comment:
Note that having to set this `margin` on the different components is usually an indication that the `margin` can be set once on the container of these components. Is that possible here?